src/Tab/Tab.tsx

```diff
@@ -1,6 +1,11 @@
-import PropTypes from 'prop-types';
+const defaultProps = {
+  defaultChecked: false,
+};
 
-function Tab({ children, defaultChecked }) {
+export default function Tab(
+  { children, defaultChecked = false }:
+  { children: string, defaultChecked?: boolean } & typeof defaultProps,
```

Comment:
Will discuss it.

Reply:
Removed typeof operator and turned off the react/require-default-props rule.